src/main/java/edu/ucla/library/bucketeer/handlers/LoadCsvHandler.java

```diff
@@ -46,6 +45,11 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(LoadCsvHandler.class, Constants.MESSAGES);
 
+    private static final String JOB_FINALIZER = FinalizeJobVerticle.class.getName();
+
+    /* Default maximum number of bytes in a source image file that we can process on AWS Lambda */
+    private static final int DEFAULT_MAX_FILE_SIZE = 300000000; // 300 MB
```

Comment:
Should this value be sourced from the POM instead?

Reply:
I'm not sure what you mean by sourced from the POM. It's defined in the POM so that this value can be overridden by whatever value someone puts into the POM: `myConfig.getInteger(Config.MAX_SOURCE_SIZE, DEFAULT_MAX_FILE_SIZE)` I'm not sure I see the need to put the fallback as a configurable value in the POM when we have a value in there already that let's someone specify the value that's used. This is just in the rare case that someone has zeroed out the default value that's in the POM.

Reply:
Ohh, I see. I overlooked that line you quoted, my bad